aws/resource_aws_vpc_endpoint.go

```diff
@@ -201,7 +205,70 @@ func resourceAwsVpcEndpointRead(d *schema.ResourceData, meta interface{}) error
 		return nil
 	}
 
-	return vpcEndpointAttributes(d, vpce.(*ec2.VpcEndpoint), conn)
```

Comment:
Replace call to `vpcEndpointAttributes()` with explicit setting of attributes (including new `requester_managed` attribute).